cdap-ui/app/features/hydratorplusplus/controllers/create/partials/nodeconfig-ctrl.js

```diff
@@ -31,6 +32,8 @@ class HydratorPlusPlusNodeConfigCtrl {
     this.ConfigStore = HydratorPlusPlusConfigStore;
     this.$scope.isDisabled = rDisabled;
     this.HydratorPlusPlusHydratorService = HydratorPlusPlusHydratorService;
+    this.myPipelineApi = myPipelineApi;
+    this.previewStore = HydratorPlusPlusPreviewStore;
```

Comment:
I think if we are sending `previewMode` as resolve parameter to the modal then I guess we could send the `previewId` too via the same. Any specific we are using the store directly here? I am ok with using entirely the store or just resolve parameters.